subsys/mgmt/Kconfig.mcumgr

```diff
@@ -52,13 +57,26 @@ menuconfig MCUMGR_CMD_IMG_MGMT
 	  Enables mcumgr handlers for image management
 
 if MCUMGR_CMD_IMG_MGMT
-	config IMG_MGMT_UL_CHUNK_SIZE
+config IMG_MGMT_UL_CHUNK_SIZE
 	int "Maximum chunk size for image uploads"
 	default 512
 	help
 	  Limits the maximum chunk size for image uploads, in bytes.  A buffer of
 	  this size gets allocated on the stack during handling of a image upload
 	  command.
+
+config IMG_MGMT_VERBOSE_ERR
+	bool "Verbose logging when uploading a new image"
+	default n
```

Comment:
`n` is the implicit default, so this line is redundant

Reply:
fixed